pzsd_bot/cogs/reminders/admin.py

```diff
@@ -0,0 +1,212 @@
+    OptionChoice(name="Japan/Korea (JST/KST)", value="Asia/Tokyo"),
+    OptionChoice(name="Bangkok (ICT)", value="Asia/Bangkok"),
+    OptionChoice(name="Hong Kong (HKT)", value="Asia/Hong_Kong"),
+    OptionChoice(name="Sydney (AEST/AEDT)", value="Australia/Sydney"),
+    OptionChoice(name="Perth (AWST)", value="Australia/Perth"),
+    OptionChoice(name="Cairo (EET/EEST)", value="Africa/Cairo"),
+    OptionChoice(name="Lagos (WAT)", value="Africa/Lagos"),
+    OptionChoice(name="Bogotá (COT)", value="America/Bogota"),
+    OptionChoice(name="Mexico City (CST/CDT)", value="America/Mexico_City"),
+    OptionChoice(name="São Paulo (BRT/BRST)", value="America/Sao_Paulo"),
+]
+
+
+class RemindersAdmin(Cog):
+    reminder_cmd = SlashCommandGroup("reminder", "Manage reminders.")
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    async def fetch_reminders(self, *args: List[BinaryExpression]) -> List[Row]:
+        async with Session.begin() as session:
+            result = await session.execute(select(reminder).where(*args))
+            reminders = result.all()
+
+        return reminders
+
+    def make_reminder_pages(self, reminder_rows: List[Row]) -> List[Embed]:
+        pages = []
+
+        for reminder_data in sorted(reminder_rows, key=lambda r: r.remind_at):
+            embed = Embed()
+            embed.description = f"### Reminder:\n{reminder_data.reminder_text}"
+
+            value = "status: **{}**\nwhen: <t:{}:R>".format(
+                reminder_data.status.value,
+                int(reminder_data.remind_at.timestamp()),
+            )
+            embed.add_field(name="Reminder info:", value=value, inline=False)
+
+            channel = self.bot.get_channel(reminder_data.channel_id)
+            if channel is not None:
+                original_message = channel.get_partial_message(
+                    reminder_data.original_message_id
+                )
+                jump_url = original_message.jump_url
+            else:
+                jump_url = None
+
+            if reminder_data.recurrence_interval is not None:
+                recurrence_duration = duration(
+                    seconds=reminder_data.recurrence_interval
+                ).in_words()
+            else:
+                recurrence_duration = None
+
+            value = (
+                f"Reminder ID: {reminder_data.id}\n"
+                f"owner: <@{reminder_data.owner}>\n"
+                f"channel: <#{reminder_data.channel_id}>\n"
+                f"original message: {jump_url}\n"
+                f"is_recurring: {reminder_data.is_recurring}\n"
+                f"recurrence_interval: {recurrence_duration}\n"
+                f"created_at: <t:{int(reminder_data.created_at.timestamp())}:f>"
+            )
+            embed.add_field(name="Metadata:", value=value, inline=False)
+
+            pages.append(embed)
+
+        return pages
+
+    @slash_command(description="Set default timezone for reminders.")
```

Comment:
that's the first use case, but it feels weird that this lives in reminders

Reply:
It's not just the primary use case, it's the only use case. But I get what you mean. Perhaps if there was a cog specific for user related commands, but there isn't one. This seems as good a place as any.